### ISSUE TITLE
Bars not showing

### ISSUE THREAD

**Issue by Kisty** [rianadon/timer-bar-card#32]
**Describe the bug**
Thanks for this card! It's really great at visualising timers. The bars don't show using the `entities` and the text is maligned. Tried using just a single entity but that didn't work either. Using Vivaldi (a Chromium-based browser) on Ubuntu 18.04 LTS x86_64

**Your card configuration**
```yaml
type: custom:timer-bar-card
entities:
  - timer.pomodoro_main
  - timer.pomodoro_planning
  - timer.pomodoro_break
  - timer.pomodoro_break_long
...
```

**Debug information**
Please add `debug: true` to your configuration and either take a screenshot or paste the output here!
![2021-12-15 09 47 03 192 168 1 100 c9f1e7ac5720](https://user-images.githubusercontent.com/3243752/146164401-7a26184b-7d80-4252-8e36-7ac20848c8b6.png)
With `debug: true`
![2021-12-15 09 56 22 192 168 1 100 45a5bc213017](https://user-images.githubusercontent.com/3243752/146164806-163a5f38-3146-4d2c-97b8-b7506e4cdfe3.png)

**Comment by rianadon** [comment]
I really have no idea. If the debug mode says `Mode: active`, there will be a progress bar.

Is the HTML element for the bar present? Or if you haven't already tried, does using a different browser / os make a difference?

**Comment by Plawasan** [comment]
Same thing for me since 2021.12.x

**Comment by aheath70** [comment]
Same thing for me since 2021.12.x

**Comment by Kisty** [comment]
Hmm, that is strange! It has only happened recently so probably the same
case for me.

Did something happen with Lovelace rendering in 2021.12.x?

**Comment by rianadon** [comment]
Alas I realized my dev environment was configured wrong and I see this too. The `hui-generic-entity-row` element, which the card depends on, was restructured the latest update which breaks the layout.

Lesson learned: I should not have depended on that element staying the same. I'm working on removing this dependency.

**Comment by rianadon** [comment]
This should be fixed in v1.15!
